CalcualtorLogic.js

```diff
@@ -18,16 +18,22 @@ class CalculatorLogic {
 };
 
 let calculatorLogic = new CalculatorLogic();
-calcualtorLogic.add();
-calcualtorLogic.subtract();
-calcualtorLogic.multiply();
-calcualtorLogic.divide();
+calculatorLogic.add();
+calculatorLogic.subtract();
+calculatorLogic.multiply();
+calculatorLogic.divide();
 
 
 class Calculator {
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
     };
-    render() {};
+    render() {
+        const wrapperDiv = document.createElement('div');
+        wrapperDiv.classList.add("wrapper");
+        // console.log(wrapperDiv.outerHTML);
```

Comment:
if you want to see the result of this console

Then you need to to create an instance of the `Calculator ` class and dont' forget to pass a CSS selector to the class 

```js
      const calc = new Calculator(".you-need-to-pass-css-selector-to-html-element-here-pls-pls")
      calc.render();


```

Reply:
yes I want 
I wanted to ask you about what you answered me

thanks